sagemaker-featurestore/sagemaker_featurestore_fraud_detection_python_sdk.ipynb

```diff
@@ -414,7 +414,7 @@
    "cell_type": "markdown",
```

Comment:
<p>We should not replace get_record with batch_get_record totally. Sample notebook should be a playground for customers to be able to try out all our APIs. We can add another cell and explain that not only you can use get_record to retrieve a single record, you can use another batch_get_record API to fetch record from multiple feature groups and recordIdentifiers. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
No problem. Updated my code.